tests/unit_test/swapanything_test/test_cli.py

```diff
@@ -0,0 +1,96 @@
+import os
+from pathlib import Path
+
+import pandas as pd
+from click.testing import CliRunner
+from swapanything.cli import HELP_MESSAGE, main
+
+
+def test_cli_help():
+    '''
+    Check that the help message is printed.
+    '''
+    runner = CliRunner()
+    result = runner.invoke(main, ['--help'])
+    assert result.exit_code == 0
+
+    # Do some string manipulation to make the output string comply with the required message
+    single_line_output = result.output.replace('\n', '').replace('  ', ' ')
+    assert HELP_MESSAGE in single_line_output
+
+
+def test_cli_compute_slots(tmp_path: Path):
+    '''
+    Check that the slot computation works with default parameters.
+    '''
+    runner = CliRunner()
+
+    # not using runner.isolated_filesystem() because it would the CWD and the input csv wouldn't be found, pytest tmp dir is more effective
+    input_csv_path = os.path.join('tests', 'unit_test', 'data', 'faker_availabilities.csv')
```

Comment:
Use a path relative to the __file__ special variable (the path of the current file). This way the execution is deterministic (does not depend on where you launch the pytest command)